### PR TITLE
Don't break modular design

### PR DESCRIPTION
**Pull request by forki**
Removes Messages.fs and makes things modular again

### CODE REVIEW

**Comment by forki** [comment]
@vaskir I already incorporated some of your slack critique points

File: src/Client/App.fs [hunk 1]
```diff
@@ -10,33 +10,34 @@ open Elmish.React
...
+
+type Msg = 
+| LoggedIn
+| LoggedOut
+| StorageFailure of exn
+| OpenLogIn
+| MenuMsg of Menu.Msg
+| LoginMsg of Login.Msg
+| WishListMsg of WishList.Msg
+| Logout
```
Comment (vasily-kirichenko) [inline src/Client/App.fs]: Why no indent here?

File: src/Client/Page.fs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+open Elmish.Browser.UrlParser
+
+/// The different pages of the application. If you add a new page, then add an entry here.
+[<RequireQualifiedAccess>]
+type Page = 
+| Home 
+| Login
+| WishList
```
Comment (vasily-kirichenko) [inline src/Client/Page.fs]: Why no indent?

File: src/Client/pages/Login.fs [hunk 3]
```diff
@@ -22,6 +21,15 @@ type Model = {
...
     ErrorMsg : string }
 
+
+/// The messages processed during login 
+type Msg =
+| GetTokenSuccess of string
+| SetUserName of string
+| SetPassword of string
+| AuthError of exn
+| ClickLogIn
```
Comment (vasily-kirichenko) [inline src/Client/pages/Login.fs]: Why no indent?

File: src/Client/pages/Menu.fs [hunk 4]
```diff
@@ -10,24 +10,37 @@ open Elmish.Browser.UrlParser
...
+
+/// The user data sent with every message.
+type UserData = 
+  { UserName : string 
+    Token : ServerCode.Domain.JWT }
 
 type Model = {
     User : UserData option
     query : string
 }
```
Comment (vasily-kirichenko) [inline src/Client/pages/Menu.fs]:
```fsharp
type Model = 
  { User : UserData option
    query : string }
```
?

File: src/Server/WebServer.fs [hunk 5]
```diff
@@ -11,7 +11,9 @@ open Suave.RequestErrors
 open Microsoft.Azure.WebJobs
 open ServerCode.Storage.AzureTable
 
-type DatabaseType = FileSystem | Azure of ConnectionString:AzureConnection
+type DatabaseType = 
+    | FileSystem 
+    | Azure of ConnectionString : AzureConnection
```
Comment (vasily-kirichenko) [inline src/Server/WebServer.fs]: Why CamelCase for the field name?

File: src/Client/pages/WishList.fs [hunk 6]
```diff
@@ -248,20 +258,20 @@ let view (model:Model) (dispatch: AppMsg -> unit) =
...
                             else
                                 yield a [ Href book.Link; Target "_blank"] [str book.Title ] ]
                         td [] [ str book.Authors ]
-                        td [] [ buttonLink "" (fun _ -> dispatch (WishListMsg (RemoveBook book))) [ str "Remove" ] ]
+                        td [] [ buttonLink "" (fun _ -> dispatch (RemoveBook book)) [ str "Remove" ] ]
                         ]
             ]
         ]
-        newBookForm (model) dispatch
+        newBookForm model dispatch
```
Comment (vasily-kirichenko) [inline src/Client/pages/WishList.fs]: 👍

File: src/Client/Page.fs (hunk 2 above)
Comment (forki) [reply src/Client/Page.fs]: good catch!

File: src/Server/WebServer.fs (hunk 5 above)
Comment (forki) [reply src/Server/WebServer.fs]: what do you suggest?

Comment (vasily-kirichenko) [reply src/Server/WebServer.fs]: `| Azure of connectionString : AzureConnection`

**Pull request merged**